src/workloads/execution/MultiDeletesBatched.yml

```diff
@@ -3,6 +3,12 @@ Owner: "@mongodb/server-execution"
 Description: >
   Deletes a large range of documents using the BATCHED_DELETE query exec stage while performing writes on another collection.
 
+Clients:
+  Default:
+    QueryOptions:
+      # Allow for longer duration since the loading of the data may take a while.
+      socketTimeoutMS: 3_600_000  # = 1 hour
```

Comment:
@aggrand could you point me to the TIG ticket for the issue you discussed earlier with @haleyConnelly, regarding the socketTimeout setting not working inside the `phases` yml files. We want to reference it here to avoid stepping on the same rake in the future.

For posterity, if I move this whole `Clients` section into src/phases/execution/MultiDeletes.yml, the mass-delete phase (which would take a little over 10 minutes) fails after exactly 5 minutes with the following message. Moving this Clients block back to src/workloads/execution/MultiDeletes* resolves the issue.
```
[2022/03/10 15:41:23.052] [37m15:41:23Z> [0m [2022-03-10 15:41:23.015737] [0x00007fa2dfa89700] [info]    Done with load phase. All 10000000 documents loaded into Collection0
[2022/03/10 15:46:23.179] [37m15:46:23Z> [0m [2022-03-10 15:46:23.143063] [0x00007fa2e028a700] [error]   Metrics not reported because operation 'Crud' being run by actor 'LargeMultiDelete0' did not close with success() or failure().
```

Reply:
I filed https://jira.mongodb.org/browse/PERF-2850